.github/workflows/release_win.yml

```diff
@@ -104,33 +103,42 @@ jobs:
       if: ${{ always() }}
       run: |
         cd onnx
-        python -m pip uninstall -y numpy onnx && python -m pip install numpy
+        python -m pip uninstall -y numpy onnx
+        python -m pip install numpy
         Get-ChildItem -Path dist/*.whl | foreach {python -m pip install --upgrade $_.fullname}
         pytest
 
     - name: Verify ONNX with the latest protobuf
       if: ${{ always() }}
       run: |
         cd onnx
-        python -m pip uninstall -y protobuf onnx && python -m pip install protobuf
+        python -m pip uninstall -y protobuf onnx
+        python -m pip install protobuf
         Get-ChildItem -Path dist/*.whl | foreach {python -m pip install --upgrade $_.fullname}
         pytest
 
     - name: Verify ONNX with the minimumly supported packages
       if: ${{ always() }}
       run: |
         cd onnx
-        python -m pip uninstall -y protobuf numpy onnx && python -m pip install -r requirements-min.txt
+        python -m pip uninstall -y protobuf numpy onnx
+        python -m pip install -r requirements-min.txt
         Get-ChildItem -Path dist/*.whl | foreach {python -m pip install --upgrade $_.fullname}
         pytest
 
     - name: Verify ONNX with ONNX Runtime PyPI package
       if: matrix.python-version != '3.12'
       run: |
         cd onnx
-        python -m pip uninstall -y protobuf numpy && python -m pip install -q -r requirements-release.txt
+        python -m pip uninstall -y protobuf numpy
+        python -m pip install -q -r requirements-release.txt
         python -m pip install -q onnxruntime==1.17.3
         $Env:ORT_MAX_IR_SUPPORTED_VERSION=9
         $Env:ORT_MAX_ML_OPSET_SUPPORTED_VERSION=3
         $Env:ORT_MAX_ONNX_OPSET_SUPPORTED_VERSION=20
         pytest
+
+    - name: Clean Workspace
+      if: always()
```

Comment:
Why is this required?

Reply:
Because it is not always guaranteed to have a clean workspace in all github instances.
One example is github enterprise :)

Reply:
sorry, somehow i missed the github notifications for this pr

Reply:
I see. check if the updated checkout step solves this? If not feel free to add the step back with comments for future readers. Thanks!